src/main/resources/static/login.js

```diff
@@ -0,0 +1,13 @@
+    var $input = $("input"),
+    $button = $("button");    
+$button.attr('disabled', true);
+
+$input.keyup(function() {
+    var trigger = false;
+    $input.each(function() {
+        if (!$(this).val()) {
```

Comment:
In general, we don't want to use a boolean expression to set a boolean constant. Consider the following rewrite.

```Javascript
trigger = trigger || !$(this).val();
```